apps/docs/src/layouts/MainLayout.astro

```diff
@@ -17,13 +17,13 @@ type Props = {
 const { frontmatter, headings } = Astro.props as Props;
 const canonicalURL = new URL(Astro.url.pathname, Astro.site);
 const currentPage = Astro.url.pathname;
-const currentFile = `src/pages${currentPage.replace(/\/$/, "")}.mdx`;
+const currentFile = `src/pages${currentPage.replace(/\/$/, '')}`;
```

Comment:
with that change neither main page `edit` works nor the components edit pages. Please fix